Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -195,7 +196,7 @@ public override void OnPostRaycast()
                         {
                             // If it's NOT a hotspot, check if the hit normal is too steep 
                             // (Hotspots override dot requirements)
-                            TeleportSurfaceResult = Vector3.Dot(Result.StartPoint, Vector3.up) < upDirectionThreshold
+                            TeleportSurfaceResult = Vector3.Dot(Result.Details.LastRaycastHit.normal, Vector3.up) > upDirectionThreshold
```

Comment:
can devs override this threshold? for example if they want to make an experience where the user climbs steep inclines?

Reply:
Yes by setting the value in the inspector.